src/components/PeopleFilters.tsx

```diff
@@ -27,66 +56,101 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
+            <SearchLink
               data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
+              className={centuries.includes('16')
+                ? 'button mr-1 is-info'
+                : 'button mr-1'}
+              params={{
+                centuries: centuries.includes('16')
+                  ? centuries.filter(centur => centur !== '16')
+                  : centuries.concat(['16']),
```

Comment:
maybe you can somehow move this to separate function to evaluate all params values you're using?